recipe/meta.yaml

```
@@ -322,6 +325,7 @@ outputs:
         -   popd
         - popd
         - test ! -f default.profraw   # [osx]
+        - python -c "import ssl; assert '{{ openssl }}' in ssl.OPENSSL_VERSION"
```

assert ssl.OPENSSL_VERSION.startswith('{{ openssl }}')`
```suggestion
        - python -c "import ssl; assert ssl.OPENSSL_VERSION.startswith('{{ openssl }}')"
```